src/main/java/frc/robot/subsystems/scoring/AimerIOTalon.java

```diff
@@ -0,0 +1,67 @@
+package frc.robot.subsystems.scoring;
+
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import frc.robot.Constants.ScoringConstants;
+
+public class AimerIOTalon implements AimerIO {
+    // TODO: Tune this later
+    private final TalonFX aimerLeft = new TalonFX(ScoringConstants.aimLeftMotorId);
+    private final TalonFX aimerRight = new TalonFX(ScoringConstants.aimRightMotorId);
+
+    private final MotionMagicVoltage controller = new MotionMagicVoltage(0).withSlot(0);
+    private final ArmFeedforward feedforward =
+            new ArmFeedforward(
+                    ScoringConstants.aimerkS,
+                    ScoringConstants.aimerkG,
+                    ScoringConstants.aimerkV,
+                    ScoringConstants.aimerkA);
+    private final Slot0Configs slot0 = new Slot0Configs();
+
+    private final DutyCycleEncoder encoder = new DutyCycleEncoder(ScoringConstants.aimEncoderPort);
+
+    double goalAngleRad = 0.0;
+
+    public AimerIOTalon() {
```

Comment:
Doesn't have to be in this PR, but we'll need to configure the trapezoidal profile / motion magic here as well